### ISSUE TITLE
How to init the backbone without pretrained file?

### ISSUE THREAD

**Issue by Levine66** [open-mmlab/mmdetection#10567]
Firstly，thanks for your work.

I recently work on my instance segementation  task in my custom dataset(COCO format). For example, I would like to train a brand new Mask-RCNN, which means there is no "load_from" in config file and no type of "Pretrained" in init_cfg  of backbone. I found that if there is no "Pretrained" in init_cfg, the loss of network would be nan after the first epoch, and the accuracy of the result is 100%.

I would like to know whether there is a way to let us to retrain a network. I have set the type in init_cfg to "Constant", "Xavier" and so on, it didn't work either.

**Comment by hhaAndroid** [comment]
please refer to https://github.com/open-mmlab/mmdetection/blob/main/configs/scratch/README.md

**Comment by Levine66** [comment]
> please refer to https://github.com/open-mmlab/mmdetection/blob/main/configs/scratch/README.md

Thanks for your reply, it works in my test. I find the key to solve this problem is to change the norm_cfg from "BN" to "GN". So I have two questions I'd like to ask you:
1.Why it works when we changed the "norm_cfg"?
2.When i use a new network config, how can i know which components i need to modify?

Thanks for your help again!

**Comment by Levine66** [comment]
The config files in scratch solve my problems.
